src/pkgcheck/checks/python.py

```diff
@@ -268,6 +274,49 @@ def desc(self):
         updates = ', '.join(self.updates)
         return f'PYTHON_COMPAT update{s} available: {updates}'
 
+class PythonOptionalCheck(Check):
+    """Check Python ebuilds for missing optional dependencies.
+
+    The problematic case for us is DISTUTILS_OPTIONAL,
+    DISTUTILS_USE_PEP517 != no, but no ${DISTUTILS_DEPS} anywhere.
```

Comment:
Please surround the code parts with 2 backticks, so it will render as code in html & man
(In case I named it wrong, backtick is the less-used symbol for bash sub-shell)

Also, the comma after `DISTUTILS_OPTIONAL` is semi hard for me to understand it's meaning. Maybe better replace with and, so it will be `a and b, but no c`?

Reply:
Good suggestion!